src/main/java/com/ttrides/turntablRides/controllers/UserController.java

```diff
@@ -1,16 +1,19 @@
 package com.ttrides.turntablRides.controllers;
 
+import com.ttrides.turntablRides.models.response.TTResponse;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 
 @RestController
-@RequestMapping("/api/v1/login")
+@RequestMapping("/api/v1")
```

Comment:
since this is a user controller lets add /user to the main endpoint. eg. /api/v1/user